### ISSUE TITLE
Can't convert doc to html

### ISSUE THREAD

**Issue by zhongjin616** [unoconv/unoserver#5]
hi, I test unoconvert  with libreoffice7.1.7

`unoconvert 97html转换文档.doc 977.html`

cause an error:
```
INFO:unoserver:Starting unoconverter.
INFO:unoserver:Opening 97html转换文档.doc
Traceback (most recent call last):
  File "/root/miniconda3/envs/docvert/bin/unoconvert", line 9, in <module>
    sys.exit(main())
  File "/root/miniconda3/envs/docvert/lib/python3.8/site-packages/unoserver/converter.py", line 246, in main
    result = converter.convert(
  File "/root/miniconda3/envs/docvert/lib/python3.8/site-packages/unoserver/converter.py", line 186, in convert
    raise RuntimeError(
RuntimeError: Could not find an export filter from com.sun.star.text.TextDocument to graphic_HTML
```

but i can do `libreoffice --headless --convert-to html html转换文档.doc` succeed.

**Comment by regebro** [comment]
Could you share the document? It doesn't seem to be a generalproblem.

**Comment by lublak** [comment]
@regebro i have the same issue with a pptx file.
Just create a simple pptx and use unoserver unoconvert somepresi.pptx somehtml.html.
Currently i can't send this file because there are some personal datas (author, last edit user etc.)

**Comment by lublak** [comment]
I can force with: `filtername = "impress_html_Export"`
But than i only get a single html file. It would be nice to support a folder export with a complete html structur.

**Comment by lublak** [comment]
@regebro 

I try to export presentation as full web pages.

It is possible to convert a presentation to a complete html page via the interface. So LibreOffice already has the possibility. Only how it looks like via the command line I do not know.

Export:
![grafik](https://user-images.githubusercontent.com/44057030/159218541-9deae27d-3668-48e8-947c-f641c9b3b07e.png)
as html
![grafik](https://user-images.githubusercontent.com/44057030/159218591-0f9449b7-7735-4f92-803b-0b1dbc1359a9.png)
Standard-HTML
![grafik](https://user-images.githubusercontent.com/44057030/159218430-e9077fdb-f0c6-44f2-8a6a-460ee6a782b6.png)
![grafik](https://user-images.githubusercontent.com/44057030/159218520-0aef91ec-4131-4dc9-a8eb-08b9fa738c40.png)

**Comment by regebro** [comment]
Yeah, that includes defining export formats, etc, which I don't know how to do. It's possible it can be done if we implement support for filter flags, but I'm not sure even then.

**Comment by felixble** [comment]
Hi @regebro,

first of all thanks a lot for your great work with this package!

We are having the same issue when using unoconvert to convert `odt` to `html`.

Executing `unoconvert --convert-to html test.odt test.html` causes the following error:

```
INFO:unoserver:Starting unoconverter.
INFO:unoserver:Opening //b577c1152d56441fa928bd54d914ee07.odt
Traceback (most recent call last):
  File "/usr/local/bin/unoconvert", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/unoserver/converter.py", line 248, in main
    inpath=args.infile, outpath=args.outfile, convert_to=args.convert_to
  File "/usr/local/lib/python3.7/dist-packages/unoserver/converter.py", line 186, in convert
    f"Could not find an export filter from {import_type} to {export_type}"
RuntimeError: Could not find an export filter from com.sun.star.text.TextDocument to graphic_HTML
```

We can do it with LibreOffice from the command line with the following command: `soffice --headless --convert-to html test.odt`.

This produces the following output on the cli: `convert /data/test.odt -> /data/test.html using filter : HTML (StarWriter)`.

It looks like there is an issue when figuring out the correct filter in https://github.com/unoconv/unoserver/blob/3e30d67387ebfa0041ec9e29a67b52ae0cd49d35/src/unoserver/converter.py#L87

Could you please have a look at this?

Thanks!

**Comment by asmundstavdahl** [comment]
Can be solved with #59
